src/rqt_pose_view/pose_view_widget.py

```diff
@@ -57,6 +57,14 @@ def __init__(self, plugin):
         loadUi(ui_file, self)
         self._plugin = plugin
 
+        # subscribe immediately if topic name found on param server
+        if rospy.has_param('~topic'):
+            topic = rospy.get_param('~topic')
+            try:
+                self._subscribe_topic(topic)
+            except(AttributeError):
+                rospy.logwarn('invalid topic name {}'.format(topic))
+
```

Comment:
I think this block needs to happen *after* all member variables have been initialized (so at the end of the `__init__` function). Otherwise e.g. the `self._topic_name` is immediately overwritten with `None` in line 70.

Reply:
That makes sense, I'll fix it.

Reply:
Fixed in 8a10a4c. I've tested the changes with a dummy node publishing some Imu messages, and it works as it should (the auto-subscribing itself and the proper setting of `self._topic_name`).